.github/workflows/ci.yaml

```
@@ -542,23 +542,35 @@ jobs:
     name: Oracle 23 integration test
     runs-on: ubuntu-latest
 
-    services:
-      oracle:
-        image: ghcr.io/scalar-labs/oracle/db-prebuilt:23
-        credentials:
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.CR_PAT }}
-        ports:
-          - 1521:1521
-        options: >-
-          --health-cmd "/opt/oracle/checkDBStatus.sh"
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 120
     steps:
-      - name: Free up ~2GB of disk space by removing .NET runtime and libraries.
+      - name: Free up ~14GB of disk space by removing the Android SDK
         run: |
-          sudo rm -r /usr/share/dotnet
+          echo "Storage available before deletion"
+          df -h /
+          echo 
+          sudo rm -r /usr/local/lib/android
+          echo "Storage available after deletion"
+          df -h /
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+
+      - name: Start Oracle 23 container
+        run: docker run -d --name oracle-23 -p 1521:1521  ghcr.io/scalar-labs/oracle/db-prebuilt:23.2.0-free
```

```suggestion
        run: docker run -d --name oracle-23 -p 1521:1521  ghcr.io/scalar-labs/oracle/db-prebuilt:23
```
Sorry, I forgot to use the tag instead of the specific version.